store/tikv/txn.go

```diff
@@ -60,6 +60,7 @@ type tikvTxn struct {
 	snapshot  *tikvSnapshot
 	us        kv.UnionStore
 	store     *tikvStore // for connection to region.
+	txnScope  string     // txn_scope indicates which txn_scope this TiKV transaction will work with.
```

Comment:
Maybe it's better to use `Option` instead of adding one more field? Many other attributes of transactions are passed by `Option`, see `kv/kv.go`.
You can also refer to the usages of `Transaction.SetOption` and `tikvTxn.us.GetOption`.

Reply:
Good idea!